Sources/ScalableProtocol.swift

```diff
@@ -2,4 +2,5 @@ import Foundation
 
 public protocol Scalable {
     func scale(withScaleFactor scaleFactor: Int)
+    func printAreaToConsole() throws
```

Comment:
Why are we placing a print function inside of a protocol focused on scalability?